<launch>
    <!-- <include file="$(find sih)/launch/drone.launch"/> -->
    <include file="$(find sih)/launch/detect_whycon.launch"/>
    <node pkg="sih" type="path_plan_whycon.py" name="position_hold" />
    <!-- <node name="pid_plot" pkg="rqt_plot" type="rqt_plot"
        args="/pitch_error/data /roll_error/data /yaw_error/data /alt_error/data" /> -->
    <!-- <node pkg="pid_tune" type="pid_tune_gui.py" name="pid_tune_gui" /> -->

</launch>